### PR TITLE
[HIPIFY][build] Look for clang in CMAKE_PREFIX_PATH

### PR DESCRIPTION
**Pull request by kzhuravl**
- It is needed for building hipify on an OS that does not have llvm and/or rocm installed (lightning ci does not install neither of those on its build bots)

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -13,12 +13,14 @@ if(HIPIFY_INCLUDE_IN_HIP_SDK)
...
 else()
+  find_package(Clang REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
   find_package(LLVM REQUIRED CONFIG PATHS ${CMAKE_PREFIX_PATH})
 
   message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}:")
-  message(STATUS "   - CMake module path: ${LLVM_CMAKE_DIR}")
-  message(STATUS "   - Include path     : ${LLVM_INCLUDE_DIRS}")
-  message(STATUS "   - Binary path      : ${LLVM_TOOLS_BINARY_DIR}")
+  message(STATUS "   - CMake module path:  ${LLVM_CMAKE_DIR}")
+  message(STATUS "   - Clang include path: ${CLANG_INCLUDE_DIRS})
```
Comment (emankov) [inline CMakeLists.txt]: Typo: the ending `"` is missing.

**Review by emankov — CHANGES_REQUESTED** [review]
LGTM, except for the typo, which breaks the build.

File: CMakeLists.txt (hunk 1 above)
Comment (kzhuravl) [reply CMakeLists.txt]: Fixed. Thanks.

**Review by emankov — APPROVED** [review]
LGTM, thank you!

**Pull request merged**